com.github.juliagoda.antimicro.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id": "com.github.juliagoda.antimicro",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "antimicro",
+    "finish-args": [
+        "--device=all",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--share=network",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--filesystem=home:rw",
+        "--socket=system-bus"
```

Comment:
Why an app would require the system-bus access?

Reply:
https://github.com/juliagoda/antimicro/issues/44

Reply:
You should build the app and run it `flatpak run --log-session-bus com.github.juliagoda.antimicro` you will be able to see which bus's you want to communicate with.